script/config/env.lua

```diff
@@ -0,0 +1,55 @@
+-- Handles loading environment arguments
+
+---ENV args are defined here.
+---- `name` is the ENV arg name
+---- `key` is the value used to index `_G` for setting the argument
+---@type { name: string, key: string }[]
+local vars = {
+    {
+        name = "LLS_CHECK_LEVEL",
+        key = "CHECKLEVEL",
+    },
+    {
+        name = "LLS_CHECK_PATH",
+        key = "CHECK",
+    },
+    {
+        name = "LLS_CONFIG_PATH",
+        key = "CONFIGPATH"
+    },
+    {
+        name = "LLS_DOC_OUT_PATH",
+        key = "DOC_OUT_PATH",
+    },
+    {
+        name = "LLS_DOC_PATH",
+        key = "DOC",
+    },
+    {
+        name = "LLS_FORCE_ACCEPT_WORKSPACE",
+        key = "FORCE_ACCEPT_WORKSPACE"
```

Comment:
Just realized, this will probably need to be cast into a `boolean` to work correctly.